filip/models/ngsi_v2/base.py

```diff
@@ -43,13 +43,13 @@ class EntityPattern(BaseModel):
     @model_validator(mode='after')
     @classmethod
     def validate_conditions(cls, values):
```

Comment:
Just notived that for `mode='after'` you can make use of `self`: https://docs.pydantic.dev/dev-v2/usage/validators/#model-validators